tools/peptideshaker/fasta_cli.xml

```diff
@@ -0,0 +1,63 @@
+<tool id="fasta_cli" name="FastaCLI" version="1.0">
+    <description>
+        Appends decoy sequences to FASTA files
+    </description>
+    <macros>
+        <import>macros_basic.xml</import>
+    </macros>
+    <requirements>
+	<requirement type="package" version="@SEARCHGUI_VERSION@">searchgui</requirement>
+	<requirement type="package" version="3.0">zip</requirement>
+    </requirements>
+    <expand macro="stdio" />
+    <command>
+<![CDATA[
+
+        ## #set $temp_stderr = "searchgui_stderr"
+        #set $bin_dir = "bin"
+
+        mkdir output;
+        cwd=`pwd`;
+        export HOME=\$cwd;
+
+        ## echo the search engines to run
+        ##echo "$search_engines_options.engines";
+        echo "DB: ${input_database.display_name} sequences: ${input_database.metadata.sequences}";
+
+        cp "${input_database}" input_database.fasta;
```

Comment:
single quotes instead of double-quotes, this applied to all text, or data parameters.